python/lsst/pex/config/registry.py

```diff
@@ -123,9 +123,10 @@ class Registry(collections.abc.Mapping):
     def __init__(self, configBaseType=Config):
         if not issubclass(configBaseType, Config):
             raise TypeError(
-                "configBaseType=%s must be a subclass of Config"
-                % _typeStr(
-                    configBaseType,
+                "configBaseType={} must be a subclass of Config".format(
```

Comment:
Why `.format()` here?

Reply:
because that's what ruff did. I will change it.